### ISSUE TITLE
sometimes won't extend selection downwards

### ISSUE THREAD

**Issue by lorenrosen** [zim-desktop-wiki/zim-desktop-wiki#2249]
Under certain circumstances the selection isn't extended when a window is scrolled downwards. This happens when zim is full-screen and there is no search bar on the bottom.

1. zim 0.75.1 and Ubuntu 20.04 (haven't tried other releases)
2. maximize the zim window
1. view a note that is longer than the window can display
3. have the viewport set so it doesn't display the bottom of the note
4. move the cursor near the bottom of the window
5. click and drag the mouse downwards so as to start and extend a region selection
6. the selection extension will stop on the last visible line, instead of scrolling down and extending it

**Expected behavior**
the viewport scrolls and the selection extends

**Environment (please complete the following information):**
 zim 0.75.1 and Ubuntu 20.04
on my system both are up-to-date regarding bug fixes

**Additional context**
Note that similar circumstances (e.g. scrolling up, or window not maximized, or search bar below the window) work as expected.

**Comment by jaap-karssenberg** [comment]
This smells like Gtk behavior, not a zim specific bug. Afraid very little we can do about such things.

**Comment by lorenrosen** [comment]
Yeah wouldn't be at all surprised if it was due to GTK. I was thinking of a conceptually simple workaround: put a one-pixel high 'window' on the bottom, just below the note. Don't know if this would be easy to implement or not. (I may try implementing it myself -- I have lots of development experience.)
    On Tuesday, January 10, 2023 at 01:12:56 AM PST, Jaap Karssenberg ***@***.***> wrote:  
 
 


This smells like Gtk behavior, not a zim specific bug. Afraid very little we can do about such things.
